geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractRedisMemoryStatsIntegrationTest.java

```diff
@@ -33,10 +33,6 @@
 public abstract class AbstractRedisMemoryStatsIntegrationTest implements RedisIntegrationTest {
 
```

Comment:
Should we remove this class altogether along with the `MemoryStatsNativeRedisAcceptanceTest` class and just have the Radish test class?

Reply:
changes made as per convo at retro (data load in test increased)